### PR TITLE
ENH: Add Python code for `DuplicateAnImage` example

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add Python code for `DuplicateAnImage` example.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
[These comments](https://github.com/InsightSoftwareConsortium/ITKExamples/pull/168#issuecomment-660411600) in PR #168 applies to this PR as well.

**Comment by jhlegarreta** [comment]
Pinging @mseng10

File: src/Core/Common/DuplicateAnImage/Code.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+randomImageSource = itk.RandomImageSource[ImageType]()
+randomImageSource.SetNumberOfWorkUnits(1)  # to produce non-random results
+
+image = randomImageSource.GetOutput()
+
+duplicator = itk.ImageDuplicator[ImageType]()
+duplicator.SetInputImage(image)
+duplicator.Update()
+
+clonedImage = duplicator.GetOutput()
```
Comment (thewtex) [inline src/Core/Common/DuplicateAnImage/Code.py:33]: Here, in would be nice to use the more Pythonic:

```
cloned = itk.image_duplicator(image)
```

File: src/Core/Common/DuplicateAnImage/Documentation.rst [hunk 2]
```diff
@@ -24,6 +24,13 @@ C++
 .. literalinclude:: Code.cxx
    :lines: 18-
 
+Python
+......
+
+.. literalinclude:: Code.py
+   :language: python
+   :lines: 1, 18-
```
Comment (thewtex) [inline src/Core/Common/DuplicateAnImage/Documentation.rst:32]: `18-` to `16-`

Comment (jhlegarreta) [reply src/Core/Common/DuplicateAnImage/Documentation.rst]: Done in f7c20d4.

File: src/Core/Common/DuplicateAnImage/Code.py (hunk 1 above)
Comment (jhlegarreta) [reply src/Core/Common/DuplicateAnImage/Code.py]: Done in f7c20d4.

**Pull request merged**